src/main/kotlin/com/sourcegraph/cody/chat/AgentChatSession.kt

```diff
@@ -177,7 +177,7 @@ private constructor(
   }
 
   override fun sendWebviewMessage(message: WebviewMessage) {
```

Comment:
note: this is enable / disable remote in particular - I guess we are ok with it

Reply:
I do not understand that comment :)

Reply:
I just pointed out where we call it and hence, where `withAgentRestartIfNeeded` is possible - all good